.github/workflows/publish-immutable-actions.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Publish Immutable Action Version'
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+      packages: write
+
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v4
+      - name: Publish
+        id: publish
+        uses: actions/publish-immutable-action@0.0.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We could update this action to default ot using the `GITHUB_TOKEN`

https://github.com/actions/github-script/blob/35b1cdd1b2c1fc704b1cd9758d10f67e833fcb02/action.yml#L11-L14